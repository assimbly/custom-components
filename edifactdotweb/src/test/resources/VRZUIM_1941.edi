UNB+UNOA:1+RAET PERSONELE SYSTEMEN:HRIS+TEST RAET ARBO UNIE+180126:0755+012618075550+++N'
UNH+BerichtID1+VRZUIM:1:941:IT:VERZ01'
BGM+VER::ARB++M1'
NAD+MR+-+ARBO UNIE++Straat1:nr1+plaats++1111LZ+NL'
COM+tel1:TE'
COM+fax1:FX'
NAD+FR++frer++straat2:nr2+plaats2++2222LZ+NL'
COM+(030) 467 85 75(tel2):TE'
COM+fax2:FX'
NAD+WG+bed122212::ARB+STICHTING NS(wg)++straatwg:nrwg+plaatswg++3333LZ+NL'
COM+wgnr:TE'
COM+wgfax:FX'
RFF+ARB:46U1'
RFF+SBI'
RFF+ANB:9999999999999'
PNA+WN+000000000+pers21908+++:ELVENq+:VANq+:UMCq'
PNA+PTR+++++:ZELST+:VAN DER'
DTM+006:19000218:102'
DTM+403:40.00:805'
DTM+401:20160801:102'
DTM+402:20160802:102'
DTM+026:20170901:102'
DTM+404:20170905:102'
DTM+405:1:802'
PDI+V(ges)'
NAD+HUA'
PCD+WAO:0'
PCD+ZKT:99'
CTA+IC+23PU   4DU+afdnaam?'
EMP++1'
RFF+ZKC:7'
RFF+REI:0'
RFF+ACT:1'
GIS+N:IDB'
GIS+N:IAA'
CIN+DIA+-'
MOA+FGW:0.00'
TAX+7+TRF'
FII+WG'
FTX++++N.V.T.'
ADR+:AWN+:straatwn:nrwn+plaatswn+4444JT+NL'
COM+telwn:TE'
COM+faxwn:FX'
ADR+:VPA+:straatverp:nrverp+plaatsverp+5verJT+NL'
COM+verpnr:TE'
COM+-:FX'
UNT+45+HISPEX0'
UNH+BerichtID2+VRZUIM:1:941:IT:VERZ01'
BGM+VER::ARB++M1'
NAD+MR+-+ARBO UNIE++Straat1:nr1+plaats++1111LZ+NL'
COM+tel1:TE'
COM+fax1:FX'
NAD+FR++frer++straat2:nr2+plaats2++2222LZ+NL'
COM+(030) 467 85 75(tel2):TE'
COM+fax2:FX'
NAD+WG+bed122212::ARB+STICHTING NS(wg)++straatwg:nrwg+plaatswg++3333LZ+NL'
COM+wgnr:TE'
COM+wgfax:FX'
RFF+ARB:46U1'
RFF+SBI'
RFF+ANB:9999999999999'
PNA+WN+000000000+pers21908+++:ELVENq++:UMCq'
PNA+PTR+++++:ZELST+'
DTM+006:19000218:102'
DTM+403:40.00:805'
DTM+401:20160801:102'
DTM+402:20160802:102'
DTM+026:20170901:102'
DTM+404:20170905:102'
DTM+405:1:802'
PDI+V(ges)'
NAD+HUA'
PCD+WAO:0'
PCD+ZKT:99'
CTA+IC+23PU   4DU+afdnaam?'
EMP++1'
RFF+ZKC:7'
RFF+REI:0'
RFF+ACT:1'
GIS+N:IDB'
GIS+N:IAA'
CIN+DIA+-'
MOA+FGW:0.00'
TAX+7+TRF'
FII+WG'
FTX++++N.V.T.'
ADR+:AWN+:straatwn:nrwn+plaatswn+4444JT+NL'
COM+telwn:TE'
COM+faxwn:FX'
ADR+:VPA+:straatverp:nrverp+plaatsverp+5verJT+NL'
COM+verpnr:TE'
COM+-:FX'
UNT+45+HISPEX0'
UNH+BerichtID3+VRZUIM:1:941:IT:VERZ01'
BGM+VER::ARB++M1'
NAD+MR+-+ARBO UNIE++Straat1:nr1+plaats++1111LZ+NL'
COM+tel1:TE'
COM+fax1:FX'
NAD+FR++frer++straat2:nr2+plaats2++2222LZ+NL'
COM+(030) 467 85 75(tel2):TE'
COM+fax2:FX'
NAD+WG+bed122212::ARB+STICHTING NS(wg)++straatwg:nrwg+plaatswg++3333LZ+NL'
COM+wgnr:TE'
COM+wgfax:FX'
RFF+ARB:46U1'
RFF+SBI'
RFF+ANB:9999999999999'
PNA+WN+000000000+pers21908+++:ELVENq++:UMCq'
PNA+PTR'
DTM+006:19000218:102'
DTM+403:40.00:805'
DTM+401:20160801:102'
DTM+402:20160802:102'
DTM+026:20170901:102'
DTM+404:20170905:102'
DTM+405:1:802'
PDI+V(ges)'
NAD+HUA'
PCD+WAO:0'
PCD+ZKT:99'
CTA+IC+23PU   4DU+afdnaam?'
EMP++1'
RFF+ZKC:7'
RFF+REI:0'
RFF+ACT:1'
GIS+N:IDB'
GIS+N:IAA'
CIN+DIA+-'
MOA+FGW:0.00'
TAX+7+TRF'
FII+WG'
FTX++++N.V.T.'
ADR+:AWN+:straatwn:nrwn:nrtoe+plaatswn+4444JT+NL'
COM+telwn:TE'
COM+faxwn:FX'
ADR+:VPA+:straatverp:nrverp+plaatsverp+5verJT+NL'
COM+verpnr:TE'
COM+-:FX'
UNT+45+HISPEX0'
UNZ+2+012618075550'
