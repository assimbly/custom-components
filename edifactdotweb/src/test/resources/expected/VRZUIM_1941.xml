<?xml version="1.0" encoding="UTF-8"?>
<edifactmessages xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                 xsi:noNamespaceSchemaLocation="file:///C:/Users/Bart/Documents/dw/raet/Medewerker.xsd">
    <raetSystemName>HRIS</raetSystemName>
    <messages>
        <message>
            <messageID>BerichtID1</messageID>
            <mutationType>M1</mutationType>
            <nadMR>
                <name>ARBO UNIE</name>
                <street>Straat1</street>
                <streetNumber>nr1</streetNumber>
                <place>plaats</place>
                <postalCode>1111LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadMR>
            <phoneMR>tel1</phoneMR>
            <nadFR>
                <name>frer</name>
                <street>straat2</street>
                <streetNumber>nr2</streetNumber>
                <place>plaats2</place>
                <postalCode>2222LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadFR>
            <phoneFR>(030) 467 85 75(tel2)</phoneFR>
            <nadWG>
                <companyCode>bed122212</companyCode>
                <name>STICHTING NS(wg)</name>
                <street>straatwg</street>
                <streetNumber>nrwg</streetNumber>
                <place>plaatswg</place>
                <postalCode>3333LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadWG>
            <phoneWG>wgnr</phoneWG>
            <wgbo>46U1</wgbo>
            <uwvNumber>9999999999999</uwvNumber>
            <employee>
                <empNumber>pers21908</empNumber>
                <lastname>ELVENq</lastname>
                <prefix>VANq</prefix>
                <initials>UMCq</initials>
                <partnerName>ZELST</partnerName>
                <prefixPartner>VAN DER</prefixPartner>
                <birthdate>19000218</birthdate>
                <gender>V(ges)</gender>
                <hoursPerWeek>40.00</hoursPerWeek>
                <employementStartDate>20160801</employementStartDate>
                <employementEndDate>20160802</employementEndDate>
            </employee>
            <phoneEMP>telwn</phoneEMP>
            <address>
                <street>straatwn</street>
                <streetNumber>nrwn</streetNumber>
                <place>plaatswn</place>
                <postalCode>4444JT</postalCode>
                <countryCode>NL</countryCode>
            </address>
            <addressIllness>
                <street>straatverp</street>
                <streetNumber>nrverp</streetNumber>
                <place>plaatsverp</place>
                <postalCode>5verJT</postalCode>
                <countryCode>NL</countryCode>
            </addressIllness>
            <phoneIllness>verpnr</phoneIllness>
            <illnessStartDate>20170901</illnessStartDate>
            <illnessMutationDate>20170905</illnessMutationDate>
            <illnessPerc>99</illnessPerc>
            <illnessCode>7</illnessCode>
            <ouCode>23PU   4DU</ouCode>
            <illnessRemark>N.V.T.</illnessRemark>
        </message>
        <message>
            <messageID>BerichtID2</messageID>
            <mutationType>M1</mutationType>
            <nadMR>
                <name>ARBO UNIE</name>
                <street>Straat1</street>
                <streetNumber>nr1</streetNumber>
                <place>plaats</place>
                <postalCode>1111LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadMR>
            <phoneMR>tel1</phoneMR>
            <nadFR>
                <name>frer</name>
                <street>straat2</street>
                <streetNumber>nr2</streetNumber>
                <place>plaats2</place>
                <postalCode>2222LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadFR>
            <phoneFR>(030) 467 85 75(tel2)</phoneFR>
            <nadWG>
                <companyCode>bed122212</companyCode>
                <name>STICHTING NS(wg)</name>
                <street>straatwg</street>
                <streetNumber>nrwg</streetNumber>
                <place>plaatswg</place>
                <postalCode>3333LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadWG>
            <phoneWG>wgnr</phoneWG>
            <wgbo>46U1</wgbo>
            <uwvNumber>9999999999999</uwvNumber>
            <employee>
                <empNumber>pers21908</empNumber>
                <lastname>ELVENq</lastname>
                <initials>UMCq</initials>
                <partnerName>ZELST</partnerName>
                <birthdate>19000218</birthdate>
                <gender>V(ges)</gender>
                <hoursPerWeek>40.00</hoursPerWeek>
                <employementStartDate>20160801</employementStartDate>
                <employementEndDate>20160802</employementEndDate>
            </employee>
            <phoneEMP>telwn</phoneEMP>
            <address>
                <street>straatwn</street>
                <streetNumber>nrwn</streetNumber>
                <place>plaatswn</place>
                <postalCode>4444JT</postalCode>
                <countryCode>NL</countryCode>
            </address>
            <addressIllness>
                <street>straatverp</street>
                <streetNumber>nrverp</streetNumber>
                <place>plaatsverp</place>
                <postalCode>5verJT</postalCode>
                <countryCode>NL</countryCode>
            </addressIllness>
            <phoneIllness>verpnr</phoneIllness>
            <illnessStartDate>20170901</illnessStartDate>
            <illnessMutationDate>20170905</illnessMutationDate>
            <illnessPerc>99</illnessPerc>
            <illnessCode>7</illnessCode>
            <ouCode>23PU   4DU</ouCode>
            <illnessRemark>N.V.T.</illnessRemark>
        </message>
        <message>
            <messageID>BerichtID3</messageID>
            <mutationType>M1</mutationType>
            <nadMR>
                <name>ARBO UNIE</name>
                <street>Straat1</street>
                <streetNumber>nr1</streetNumber>
                <place>plaats</place>
                <postalCode>1111LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadMR>
            <phoneMR>tel1</phoneMR>
            <nadFR>
                <name>frer</name>
                <street>straat2</street>
                <streetNumber>nr2</streetNumber>
                <place>plaats2</place>
                <postalCode>2222LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadFR>
            <phoneFR>(030) 467 85 75(tel2)</phoneFR>
            <nadWG>
                <companyCode>bed122212</companyCode>
                <name>STICHTING NS(wg)</name>
                <street>straatwg</street>
                <streetNumber>nrwg</streetNumber>
                <place>plaatswg</place>
                <postalCode>3333LZ</postalCode>
                <countryCode>NL</countryCode>
            </nadWG>
            <phoneWG>wgnr</phoneWG>
            <wgbo>46U1</wgbo>
            <uwvNumber>9999999999999</uwvNumber>
            <employee>
                <empNumber>pers21908</empNumber>
                <lastname>ELVENq</lastname>
                <initials>UMCq</initials>
                <birthdate>19000218</birthdate>
                <gender>V(ges)</gender>
                <hoursPerWeek>40.00</hoursPerWeek>
                <employementStartDate>20160801</employementStartDate>
                <employementEndDate>20160802</employementEndDate>
            </employee>
            <phoneEMP>telwn</phoneEMP>
            <address>
                <street>straatwn</street>
                <streetNumber>nrwn</streetNumber>
                <streetNumberAdd>nrtoe</streetNumberAdd>
                <place>plaatswn</place>
                <postalCode>4444JT</postalCode>
                <countryCode>NL</countryCode>
            </address>
            <addressIllness>
                <street>straatverp</street>
                <streetNumber>nrverp</streetNumber>
                <place>plaatsverp</place>
                <postalCode>5verJT</postalCode>
                <countryCode>NL</countryCode>
            </addressIllness>
            <phoneIllness>verpnr</phoneIllness>
            <illnessStartDate>20170901</illnessStartDate>
            <illnessMutationDate>20170905</illnessMutationDate>
            <illnessPerc>99</illnessPerc>
            <illnessCode>7</illnessCode>
            <ouCode>23PU   4DU</ouCode>
            <illnessRemark>N.V.T.</illnessRemark>
        </message>
    </messages>
</edifactmessages>